plat/nvidia/tegra/common/tegra_sip_calls.c

```diff
@@ -38,7 +38,14 @@
 #include <runtime_svc.h>
 #include <tegra_private.h>
 
+#define NS_SWITCH_AARCH32	1
+#define SCR_RW_BITPOS		10
```

Comment:
You could define this in terms of the SCR_RW_BIT definition by using a GCC builtin function:
`#define SCR_RW_BITPOS __builtin_ctz(SCR_RW_BIT)`
This is the technique used in the GIC drivers to implement the `interrupt_type_to_line()` functionality. (I'm actually wondering if we should have a less obscure macro to do this?